README.md

```diff
@@ -187,10 +187,13 @@
 * **[awesome-mcp-servers](https://github.com/punkpeye/awesome-mcp-servers)**: A curated list of awesome Model Context Protocol (MCP) servers. ![Stars](https://img.shields.io/github/stars/punkpeye/awesome-mcp-servers.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/punkpeye/awesome-mcp-servers?color=green) ![LastCommit](https://img.shields.io/github/last-commit/punkpeye/awesome-mcp-servers?color=green)
 * **[mcp-directory](https://github.com/chatmcp/mcp-directory)**: A directory for Awesome MCP Servers. ![Stars](https://img.shields.io/github/stars/chatmcp/mcp-directory.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/chatmcp/mcp-directory?color=green) ![LastCommit](https://img.shields.io/github/last-commit/chatmcp/mcp-directory?color=green)
 * **[Smithery](https://smithery.ai)**: Smithery is a platform to help developers find and ship language model extensions compatible with the Model Context Protocol Specification.
+* **[MCP Marketplace](https://github.com/cline/mcp-marketplace)**: This is the official repository for submitting MCP servers to be included in Cline's MCP Marketplace
+* **[MCP.so](https://mcp.so/)**: Find Awesome MCP Servers and Clients.
```

Comment:
mcp.so is the mcp-directory actually, let's remove it. Same as below.

Reply:
removed